apps/wakunode2/app.nim

```diff
@@ -467,7 +531,7 @@ proc initNode(conf: WakuNodeConf,
 
 proc setupWakuNode*(app: var App): AppResult[void] =
   ## Waku node
-  let initNodeRes = initNode(app.conf, app.rng, app.peerStore, app.dynamicBootstrapNodes)
+  let initNodeRes = initNode(app.conf, app.netConf, app.rng, app.nodeKey, app.record, app.peerStore, app.dynamicBootstrapNodes)
```

Comment:
This comment doesn't add nothing new from what I already mentioned :)
However, I believe we could generate the `netConf`, `nodeKey` and `record` inside the `initNode` with the information provided by the `app.conf` param.

Reply:
Okay got it! What if we try to do something similar that is done with the rest server?
I mean, start the `discv5` service in https://github.com/waku-org/nwaku/blob/a44d4bfbcd0d67bd5920fc85091d2feee8b5a676/apps/wakunode2/app.nim#L772 ?

I don't have strong opinion about renaming the `setupWakuNode`. Both cases can be correct :)